packages/main/src/plugin/extension/extension-loader.ts

```diff
@@ -431,6 +431,29 @@ export class ExtensionLoader {
     // now we have all extensions, we can load them
     await this.loadExtensions(analyzedExtensions);
 
+    // START OF TELEMETRY
```

Comment:
suggestion: please remove this kind of comments

// START
block
// END

usually these comments will be lost/moved across refactoring

replace it by a method that will manage the telemetry like

this.sendTelemetryProviderInfos()